src/chat/chat_panel.rs

```diff
@@ -50,11 +51,67 @@ live_design! {
     }
 
     UserChatLine = <ChatLine> {
+        margin: {left: 100}
```

Comment:
Makes the message bubble of the user look more in the "right" by indenting it more than the assistant message bubble.

Ideally, the bubble should fit the text and be aligned to the right but it's not currently trivial to implement.